server/broadcast_test.go

```diff
@@ -405,13 +559,12 @@ func TestRefreshSessions(t *testing.T) {
 	assert.Equal(bsm.sessList()[1], sess4)
 
 	// asserting that refreshes stop while another is in-flight
-	bsm.createSessions = func() ([]*BroadcastSession, error) {
-		return []*BroadcastSession{StubBroadcastSession("5"), StubBroadcastSession("6")}, nil
-	}
+	delete(bsm.sessMap, "transcoder3")
```

Comment:
Shouldn't this test be checking that the length of sessMap remains unchanged even when refreshing would introduce new sessions (i.e. sessions 5 and 6) because there is already an in-flight session? We would just want to have the refresh introduce new sessions as opposed to changing the current sessions in sessMap.

Reply:
fixed in 2c78fb9a784ea81c42a9bdbd0d91c71ab6fc255b